specs/Deployment/BuildInformation.md

```diff
@@ -0,0 +1,226 @@
+- [1. Background](#1-background)
+- [2. Examples](#2-examples)
+  - [2.1. Examples - C++](#21-examples---c)
+  - [2.2. Examples - C](#22-examples---c)
+- [3. API Details](#3-api-details)
+  - [3.1. WindowsAppSDK-VersionInfo.h](#31-windowsappsdk-versioninfoh)
+  - [3.2. WindowsAppSDK-VersionInfo.cs](#32-windowsappsdk-versioninfocs)
+  - [3.3. WindowsAppSDK-VersionInfo.json](#33-windowsappsdk-versioninfojson)
+  - [3.4. WindowsAppSDK-VersionInfo.xml](#34-windowsappsdk-versioninfoxml)
+
+# 1. Background
+
+Windows App SDK supports deployment as a set of MSIX packages for use at runtime, but some
+operations and APIs require build information about the release. This spec defines public symbols
+provided so developers can easily use Windows App SDK APIs at runtime referencing the version of
+Windows App SDK they use at build-time.
+
+For example,  `MddBootstrapInitialize(majorMinorRelease, versionTag, minVersion)` requires...
+
+* Major.Minor version of the release
+* Version tag (if any)
+* Minimum required version of MSIX packages
+
+If a developer builds an unpackaged app with Windows App SDK 1.0 Experimental1 with MSIX package
+version of 0.218.840.0, they first must initialize Bootstrap API with this information. This is
+accomplished with the call
+
+```c++
+const uint32_t c_majorMinorRelease{ (1u << 16) | 0 };
+auto c_versionTag{ "experimental1" };
+const PACKAGE_VERSION c_minVersion{ (0LLU << 48) | (218LLU << 32) | (840LLU << 16) | 0 };
+THROW_IF_FAILED(MddBootstrapInitialize(c_majorMinorRelease, c_versionTag, minVersion));
+```
+
+And each time the WinAppSDK used by the build is updated this must be changed to match. This spec
+allows developers to use constants provided by WinAppSDK ensuring the application's runtime use of
+WinAppSDK's build information always matches the WinAppSDK used at build-time (without manual
+developer intervention):
+
+```c++
+const uint32_t c_majorMinorRelease{ WINDOWSAPPSDK_RELEASE_MAJOR_MINOR };
+auto c_versionTag{ WINDOWSAPPSDK_RELEASE_VERSION_TAG_W };
+const PACKAGE_VERSION c_minVersion{ WINDOWSAPPSDK_MSIX_VERSION_UINT64 };
```

Comment:
The version _is_ MSIX specific, or at least, how we express WinAppRuntime's version in MSIX (we use different values in other tech, e.g. NuGet). If WinAppSDK was used in an AppLocal way you'd have the WinAppSDK runtime (bunch of files with your app) but there'd be no MSIX package and nowhere would this UInt64 value exist.

If you're still comfortable with Runtime instead of MSIX then sure, we can roll with that

Reply:
In my mind, this is the "version of the Windows App Runtime your app wants." MSIX is an implementation detail.  I'd go with "Runtime" as the magic token here, rather than MSIX. I wouldn't block on it, though, because in 1.1 you're going to add that single-like WinAppSDKInitialize() which hides all these constants behind a generated method. :)